R/coarse_classification.R

```diff
@@ -1,14 +1,15 @@
-#' Use a lower scale classification
+#' Use a Lower Scale Classification
 #'
 #' @description
 #' `coarse_classification()` reduces the 10 scales from `classification()`
 #' to the 4 scales below.
-#' - `+` **positive effect**: reference < lcl
-#' - `~` **no effect**: min(threshold) < lcl < reference and reference < ucl < max(threshold)
-#' - `-` **negative effect**: ucl < reference
-#' - `?` **unknown effect**: lcl < min(threshold) or max(threshold) < ucl
+#' - `+` **positive effect**: `reference < lcl`
+#' - `~` **no effect**: `min(threshold) < lcl < reference` and
+#' `reference < ucl < max(threshold)`
+#' - `-` **negative effect**: `ucl < reference`
+#' - `?` **unknown effect**: `lcl < min(threshold)` or `max(threshold) < ucl`
 #'
-#' `simplify_classification()` reduces the 6 scales from
+#' `coarse_classification()` reduces the 6 scales from
```

Comment:
Should this be `remove_sign(coarse_classification())`?

Reply:
Clarified in 3c85d7e